src/oc_bifrost/rebar.config

```diff
@@ -20,7 +20,7 @@
     {edown, ".*",
         {git, "git://github.com/uwiger/edown.git",                    {branch, "master"}}},
     {ej, ".*",
-        {git, "git://github.com/seth/ej.git",                         {branch, "master"}}},
+        {git, "git://github.com/chef/ej.git",                         {branch, "lbaker/fixes-for-erlang19"}}},
```

Comment:
Should this be merged? If not, is there an issue to get this merged and update deps?

Reply:
https://github.com/chef/chef-server/issues/1822